js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1701,7 +1737,7 @@ const EditingController = modules.ViewController.inherit((function() {
 
                 if(isError) {
                     if(editData) {
-                        editData.error = arg;
+                        this._addInternalData({ key: editData.key }).error = arg;
```

Comment:
use `this._addInternalData({ key: editData.key, error: arg })` to add new param to internal data